src/rust/jsg/v8.rs

```diff
@@ -0,0 +1,440 @@
+use core::ffi::c_void;
+use std::marker::PhantomData;
+
+use crate::Lock;
+
+#[expect(clippy::missing_safety_doc)]
+#[cxx::bridge(namespace = "workerd::rust::jsg")]
+pub mod ffi {
+    #[derive(Debug)]
+    struct Local {
+        ptr: usize,
+    }
```

Comment:
It's a shame not to have an associated lifetime here. Do you think we could integrate one?

Reply:
Ah, I see what you mean in that this is just a temporary wrapper struct.